lib/link_expansion/hashed_edition.rb

```diff
@@ -0,0 +1,37 @@
+module LinkExpansion::HashedEdition
```

Comment:
Cool, I think maybe you forgot to remove `hashed_edition.rb` from the commit as it's still showing up in GitHub.